spec/factories/categories.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+FactoryBot.define do
+  factory :category do
+    title { 'MyString' }
+    description { 'MyText' }
+    icon_url { 'MyString' }
```

Comment:
Use Faker to fill out these fields. Also we might need to use `find_or_create` method to initialize instances of `Category`. See [FactoryBot docs > getting started > custom construction](https://github.com/thoughtbot/factory_bot/blob/master/GETTING_STARTED.md#custom-construction)